### ISSUE TITLE
How did the performances differ?

### ISSUE THREAD

**Issue by alphamupsiomega** [shaohua0116/Activation-Visualization-Histogram#1]
Nice to show the difference in activations, but how did the training times, losses, validation accuracies, etc differ?

**Comment by shaohua0116** [comment]
The training loss/accuracy can be viewed on Tensorboard and the sampled testing accuracy is shown per 1000 iterations. Also, the training log shows how many instances are processed per second as well as how many seconds are needed to process a batch. 

Adding accumulative training time should be trivial. Obtaining validation accuracy can also be done by testing the saved checkpoints. I recommend you to work on these first. I may also spend some time on them once I am free. Thanks!

**Comment by alphamupsiomega** [comment]
Yes I know it is possible to do this in Tensorboard, but I'm wondering if there were significant differences in your training times, losses, accuracies for inclusion in the README, before I burn my own gpu time, since you already did this.  (This is not an issue, but just a question).

**Comment by shaohua0116** [comment]
Thanks for your explanation. I didn't take a careful look at the testing results since the main motivation of this implementation is to visualize the distributions of activations. As I remember, there's no obvious difference in term of accuracy. Also, an informative discussion of the efficiency of SELU can be found here on [Reddit](https://www.reddit.com/r/MachineLearning/comments/6g5tg1/r_selfnormalizing_neural_networks_improved_elu/).
